### ISSUE TITLE
Padding=2 in Resnet blocks?

### ISSUE THREAD

**Issue by LynnHo** [vanhuyz/CycleGAN-TensorFlow#14]
Thanks for your good code! May I ask why the padding=2 in your resnet block (which was shaved latter)? Will it make a better results?

**Comment by vanhuyz** [comment]
That’s a good point! The authors used ShaveImage at first and it got me confused. I didn't realize that ShaveImage has been removed in original code:
https://github.com/junyanz/CycleGAN/commit/ad98146d0e20240fa53f3338dbacfdd331155b49

So padding should be 1 and no need to shave, right?

**Comment by LynnHo** [comment]
Yes, it seems you can use 1 padding and remove the shaving. Another question, in your code, the batch normalization is used without scale=True, which is also different from the author's code.

**Comment by vanhuyz** [comment]
Thanks! I'll fix it.
